src/hypermedia-constants.js

```diff
@@ -347,7 +350,14 @@ export const Classes = {
 		checked: 'checked',
 		password: 'password',
 		notificationEmail: 'notificationEmail',
-		autoSetGraded: 'auto-set-graded'
+		autoSetGraded: 'auto-set-graded',
+		timing: {
+			recommended: 'recommended',
+			enforced: 'enforced',
+			continueWorking: 'continueworking',
```

Comment:
could you help me rename this to `automaticZero: 'automatic-zero'` instead of `continueWorking: 'continueworking'`
Thanks!